communication/mavlink_stream.hpp

```diff
@@ -63,12 +63,20 @@ extern "C"
 
 #define MAVLINK_BASE_STATION_ID 255
 
+/* Forward declaration for friend function */
+class Onboard_parameters;
+class Central_data;
```

Comment:
For which function do you need to declare this friend class (Central_data)?
(For Onboard_parameters, it is rather clear)